content/english/pygame-pong/activity-3.md

```diff
@@ -6,17 +6,17 @@ draft: false
 ---
 
 # Activity 3
-Kudos! You not only fixed the game boundary, you are able to give color to objects too. Now let's see if you can change the keys for paddle movement. 
+Kudos! You not only fixed the game boundary, but you are able to give color to objects too. Now, let's see if you can change the keys for paddle movement.
 
-To do this, we need to modify one of the parameters we pass to create the left paddle object. Remember, from basics section how you defined the `Paddle` class with UP and DOWN key assignments as its attributes. 
+To do this, we need to modify one of the parameters we passed in when creating the left paddle object. Remember from basics section, you defined the `Paddle` class with UP and DOWN key assignments as its attributes. 
 
 Find `// TODO (ACTIVITY 3)` in the code. Modify the appropriate attribute value for the left paddle so that pressing the 's' key moves the left paddle down. Notice that currently the downward movement of the left paddle is controlled by the 't' key. 
```

Comment:
```suggestion
Find `// TODO (ACTIVITY 3)` in the code. Modify the appropriate attribute value for the left paddle so that pressing the `'s'` key moves the left paddle down. Notice that currently the downward movement of the left paddle is controlled by the `'t'` key. 
```